packages/thunderstore-api/src/fetch/teamAddMember.ts

```diff
@@ -0,0 +1,28 @@
+import { RequestConfig } from "../index";
```

Comment:
This comes a bit late since `teamCreate.ts` already exists, but my original idea was that the `fetch` directory here would be for stuff that fetches data (not stuff that uses `fetch` function). Now the files like `teamAddMember` and `teamMembers` are a bit confusingly named. I wouldn't mind splitting the files to e.g. `fetch`, `submit` and `action` directories. Not a blocker now since refactoring should be rather easy as long as everything is exported via `src/index.ts`.